### ISSUE TITLE
How exactly do I create the labels?

### ISSUE THREAD

**Issue by mhirsc** [yeephycho/tensorflow_input_image_by_tfrecord#7]
How exactly do I create a labels file? I know what should go in it (valid labels, line by line), but what do I need to name it and where do I store it? At this point I have 6 folders -- "Bullfrogs," "Salamanders," "Goldfish," "Bullfrogs_t" (testing images of bullfrogs), "Salamanders_t" and "Goldfish_t". I want to create a classification algorithm in TensorFlow (don't actually need it, just practicing for when I have my real data). Can you help me? Thanks so much for creating this script.

**Comment by yeephycho** [comment]
Under this solution, folder name will be used as labels and each folder name will be assigned an integer (start from zero) as the softmax output label. You can try [this](https://github.com/yeephycho/nasnet-tensorflow) to get some intuition.
